code/client/clrcore/External/WeaponComponent.cs

```diff
@@ -600,7 +683,10 @@ private static ComponentAttachmentPoint _GetAttachmentPoint(WeaponHash hash, Wea
 						case WeaponComponentHash.AssaultRifleVarmodLuxe:
 							return ComponentAttachmentPoint.GunRoot;
 
-						default:
+						case WeaponComponentHash.GunrunningMk2Upgrade:
+							return ComponentAttachmentPoint.GunGripR;
+
+                        default:
```

Comment:
scary spaces 😱

Reply:
Missed that one 😄

Reply:
Or nevermind, I don't get it lol

Reply:
Would you want me to remove the spaces?

Reply:
tabs are being used everywhere, then suddenly there... before `default`...

Reply:
Ohhh, I didn't notice that, I'll change that then

Reply:
With that I now also enabled the view whitespace option in VS so I won't make that mistake again